collect_app/src/main/java/org/odk/collect/android/widgets/ExStringWidget.java

```diff
@@ -51,47 +51,6 @@
 import static android.content.Intent.ACTION_SENDTO;
 import static org.odk.collect.android.utilities.ApplicationConstants.RequestCodes;
 
-/**
```

Comment:
The rest of the app is definitely under-commented but I don't think that's a reason to remove existing comments! I would tend to leave the whole thing but it's true that it feels unusual to have user-facing documentation here.

That said, I don't believe that the translation forms to override button text are documented anywhere else, mostly because they're unofficial and not supported by XLSForm. I think leaving it here is the best option we have for the time being.